src/Himbeer/HideCommands/Main.php

```diff
@@ -22,19 +22,13 @@ class Main extends PluginBase implements Listener {
 	public function onEnable(): void {
 		$this->saveDefaultConfig();
 
-		switch ($this->getConfig()->get("mode")) {
-			case "whitelist":
-				$this->mode = self::MODE_WHITELIST;
-				break;
-			case "blacklist":
-				$this->mode = self::MODE_BLACKLIST;
-				break;
-			default:
-				$this->getLogger()->error('Invalid mode selected, must be either "blacklist" or "whitelist"! Disabling...');
-				return;
-		}
+		match ($this->getConfig()->get("mode", "whitelist")) {
+			"whitelist" => $this->mode = self::MODE_WHITELIST,
+			"blacklist" => $this->mode = self::MODE_BLACKLIST,
+			default => $this->getLogger()->error('Invalid mode selected, must be either "blacklist" or "whitelist"! Disabling...')
```

Comment:
The "return" from the previous code is missing here, so the listener would still be registered